src/sass/app/modules/reports.scss

```diff
@@ -1,3 +1,11 @@
+// Base
+@import '../../app/_common/base/reset';
```

Comment:
may I know why do we have to import base here? we can import `reports.scss` inside index scss where the base is already imported?

Reply:
This was to extract `reports.scss` and all report related stylesheets out of `app.css`. Since we default to trader, no need for clients to get the stylesheets for something that they don't want.